content/en/blog/2020-10-30-charts-repo-deprecation.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Helm Chart Repository Deprecation Update"
+slug: "charts-repo-deprecation"
+authorname: "Vic Iglesias"
+author: "@vicnastea"
+authorlink: "https://twitter.com/vicnastea"
+date: "2020-10-30"
+---
+
+Back in 2019, when the Helm v2 support timeline and end of life plan was announced, [the deprecation](https://github.com/helm/charts#deprecation-timeline) of the [helm/charts GitHub repository](https://github.com/helm/charts) was announced, as well. The primary reason for the deprecation is the significant increase in upkeep for the [repo maintainers](https://github.com/helm/charts/blob/master/OWNERS). Over the last couple of years the number of charts under maintainance increased from ~100 to 300+ causing a commesurate increase in pull requests and updates to the repo, unfortunately despite many efforts to automate review and maintainance tasks the amount of time available from maintainers has not increased in unison.
```

Comment:
> causing a commesurate increase in pull requests and updates to the repo, unfortunately despite many efforts to automate review and maintainance tasks the amount of time available from maintainers has not increased in unison.

causing a commensurate increase in pull requests and updates to the repo. Unfortunately, despite many efforts to automate review and maintenance tasks, the amount of time available from maintainers has not increased.

(corrected some typos/issues)